src/methods/get-ada-handle.ts

```diff
@@ -0,0 +1,21 @@
+import { prepareMessage } from '../utils/message.js';
+import { blockfrostAPI } from '../utils/blockfrost-api.js';
+import { MessageId } from '../types/message.js';
+import { BlockfrostServerError } from '@blockfrost/blockfrost-js';
+
+const policyID = 'f0ff48bbb7bbe9d59a40f1ce90e9e9d0ff5002ec48f232b49ca0fb9a';
+
+export default async (id: MessageId, clientId: string, name: string): Promise<string> => {
+  let data: { address: string; quantity: string }[];
+
+  try {
+    data = await blockfrostAPI.assetsAddresses(
```

Comment:
And what in case of error of the Ada Handle issuer?
Returning just an address means to trust a third party company: in case the issuer has an error and it double mint the same handle, returning just the fist address may mean (for the wallet user) to send money **to the wrong address**.
So, if we want to return just a single address, I'd say **we must check** it is the only one, and, in negative case, we need to throw a specific error.
As far as I say `blockfrost-websocket-link` does not define any custom error, but just relays on error thrown by other Blockfrost packages: could you please suggest how to create a new error specific for the double minted handle respecting Blockfrost standards?
Thank you

Reply:
If there is an erroneous mint or ADA handle is hacked and someone mints the asset again then returning first address is the safe solution as that is the original address of the handle owner. 
Alternatively we could throw an error if there are multiple addresses (wslink link can already throw and forward to the client any Error instance. Simple throw Error(“ADA handle hacked”) should worl just fine). Both are doable, the latter has disadvantage pf breaking ADA handle functionality the moment there is a second mint. 
But this is like the worst case scenario where ADA handles are doomed. 

Anyways, there is no scenario where returning multiple addresses is legit with current ADA handle standard.  Returning single address make sense and saves the additional overhead for the wslink clients.

Reply:
699264d should solve this